objects/src/accounts/storage/mod.rs

```diff
@@ -241,10 +241,15 @@ impl AccountStorage {
     }
 
     /// Returns the storage maps for this storage.
-    pub fn maps(&self) -> &[StorageMap] {
+    pub fn maps(&self) -> &BTreeMap<u8, StorageMap> {
         &self.maps
     }
```

Comment:
Where do we use this method?

Reply:
When we add the account to the advice provider

https://github.com/0xPolygonMiden/miden-base/blob/727916915ddb9a237ffb975a1f7cc2f9f69139dc/miden-lib/src/transaction/inputs.rs#L225-L233